tools/generateIndices.js

```diff
@@ -0,0 +1,82 @@
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+const posix = path.posix;
+
+const contents = [`
+/**
+ * THIS IS A GENERATED FILE
+ *
+ * Please see /tools/generateIndices.js to regenerate after structural changes or component additions / deletions
+ */
+
+`];
+const exportItems = [];
+
+const src = path.resolve(__dirname, '..', 'src');
+
+const appendExports = (dir, subDirs, files) => {
```

Comment:
Webpack isn't involved in this library anymore, it's all just Babel now. I wouldn't be against re-introducing it, just need to figure out exactly how to do that.

The original reason I moved to Babel was to keep 1:1 source:transpiled parity, since the primary consumption use case was to delve into the subdirectories to find exact files. But if we move to consuming components directly from the library root, I guess we could switch to webpack for distribution builds. Worth looking into again.